### PR TITLE
Test 90 In usr list page I should be able to change in bulk users's posittion and team

### PR DESCRIPTION
**Pull request by kristiyanplpeev**
Front end added new view for changing users team and position.

Back end added endpoints for getting teams, positions, user's team, user's position, and changint user's team and position.

### CODE REVIEW

File: client/src/services/UserService.ts [hunk 1]
```diff
@@ -13,14 +13,67 @@ import { TYPES } from "inversify/types";
...
-      "Content-Type": "application/json",
+      "Content-Type": applicationJSON,
       // eslint-disable-next-line prettier/prettier
       Authorization: `Bearer ${this.authService.getToken()}`,
     };
     return (await axios.get(`${BASE_URL}users`, { headers })).data;
   };
+
+  getUsersByIds = async (usersIds: string): Promise<Array<IUserWithTeamAndPosition>> => {
+    const headers = {
```
Comment (MilenIvanovv) [inline client/src/services/UserService.ts:26]: There is a code duplication in the service. You can think of a way how we can avoid it.

File: server/src/users/users.service.spec.ts [hunk 2]
```diff
@@ -1,14 +1,107 @@
...
+import { User } from '../model/user.entity';
+import { Teams } from '../model/teams.entity';
+import { Positions } from '../model/positions.entity';
+import { mockedUser } from '../common/holidaysMockedData';
+import { PositionsEnum, TeamsEnum } from '../common/constants';
 
 describe('UsersService', () => {
   let service: UsersService;
 
+  const mockedUserWithTeamAndPositionAsString = [
```
Comment (MilenIvanovv) [inline server/src/users/users.service.spec.ts:14]: Those objects seem similar. Think of a way to remove the duplicated code

File: server/src/users/users.service.ts [hunk 3]
```diff
@@ -1,13 +1,108 @@
...
 
-  public async getAllUsers(): Promise<Array<User>> {
-    return await this.userRepo.find();
+  setUsersTeamsAndPositions(
+    users: Array<User>,
+  ): Array<UserDetailsWithTeamAndPosition> {
+    return users.reduce((acc, el) => {
+      let team = '';
+      let position = '';
+      if (el.team === null) {
```
Comment (MilenIvanovv) [inline server/src/users/users.service.ts:24]: This is perfectly fine and readable. I would have done it like so:
`const team = el.team?.team ||  TeamsEnum.noTeam`

**Review by MilenIvanovv — CHANGES_REQUESTED** [review]
Look great to me. I left some comments where it can be improved

File: client/src/services/UserService.ts (hunk 1 above)
Comment (kristiyanplpeev) [reply client/src/services/UserService.ts:26]: done.

File: server/src/users/users.service.spec.ts (hunk 2 above)
Comment (kristiyanplpeev) [reply server/src/users/users.service.spec.ts:14]: done.

File: server/src/users/users.service.ts (hunk 3 above)
Comment (kristiyanplpeev) [reply server/src/users/users.service.ts:24]: done.

**Review by MilenIvanovv — APPROVED** [review]
LGTM

**Pull request merged**